truss/secretsManager.go

```diff
@@ -182,10 +182,16 @@ func (m SecretsManager) Vault(secret SecretConfig) (*VaultCmd, error) {
 	return Vault(kubectl, m.VaultAuth), nil
 }
 
+// Exists determins whether the local secrets file exists
+func (m SecretsManager) Exists(secret SecretConfig) bool {
+	_, err := os.Stat(secret.FilePath)
+
+	return err == nil
+}
+
 // GetDecryptedFromDisk returns the encrypted yaml configuration from disk
 func (m SecretsManager) GetDecryptedFromDisk(vault *VaultCmd, secret SecretConfig) ([]byte, error) {
-	_, err := os.Stat(secret.FilePath)
-	if err != nil {
+	if !m.Exists(secret) {
 		return []byte("secrets: {}"), nil
```

Comment:
This seems like it was causing the initial confusion, but, this https://github.com/instructure-bridge/truss-cli/pull/19/files#diff-404062eb71336a60bdd8ec16387e9db6R27 should short circuit this path with a nicer error message, right?

Reply:
It gets short-circuited here: https://github.com/instructure-bridge/truss-cli/pull/19/files#diff-404062eb71336a60bdd8ec16387e9db6R27

Reply:
Gah, that's the same link... lol Yes, this line simply utilizes the helper method. But in the viewCmd, it gets intercepted.